graph/telemetry/istio/istio.go

```diff
@@ -129,7 +129,6 @@ func buildNamespaceTrafficMap(ctx context.Context, namespaceInfo graph.Namespace
 	if o.IncludeIdleEdges {
 		idleCondition = ""
 	}
-	promApi := globalInfo.PromClient.API()
```

Comment:
is this change by a perf reason getting API on demand?

Reply:
No, it was just part of a bunch of refactors having to do with globalInfo.  To be honest, I'm glad you pointed this out.  It's better this way, because we don't know what `.API()` actually does, so we shouldn't call it repeatedly.  I'll change it back.